en/user_guide/user.rst

```diff
@@ -0,0 +1,41 @@
+User, Group and Role
+====================
+
+Open Orchestra offers the possibility to manage different users with different rights.
+
+User Management
+---------------
+
+The users page allows to search, create, edit and delete current site users.
+
+At the creation, some fields are required :
+
+* first Name
+* last Name
+* e-mail
+* username
+* password
+
+Once the user is created, you can add some information regarding his default language and his group.
+
+Group Management
+----------------
+
+In Open Orchestra, users can be members of groups.
+A group gives a user some rights and access for website.
+Each group has a name and is associated to only one website.
```

Comment:
Each group has a name, can be associated with only one website and contains a list of roles.